### PR TITLE
Tests

### PR DESCRIPTION
**Pull request by ColinWake**
Added tests and fixed usages of Review POJO

### CODE REVIEW

File: src/main/resources/templates/categories.html [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<html xmlns="http://www.w3.org/1999/xhtml" xmlns:th="https://www.thymeleaf.org" lang="en">
```
Comment (ndought) [inline src/main/resources/templates/categories.html:2]: what the heck is up with this

**Review by ColinWake — COMMENTED** [review]
Thymeleaf xml namespace removed from template, possibly needs re-added

**Review by fpemba — CHANGES_REQUESTED** [review]
add thymeleaf

**Review by ndought — APPROVED** [review]

**Review by fpemba — APPROVED** [review]
good job!

**Review by AbdulJ23 — APPROVED** [review]
cool

File: src/main/resources/templates/categories.html (hunk 1 above)
Comment (ColinWake) [reply src/main/resources/templates/categories.html:2]: Fixed with https://github.com/2020-Spring-Cohort/reviews-site-models/pull/11/commits/9e8714340cde9ea75a0862e996807b0abe75cf4e

**Pull request merged**